lib/smart_app_launch/smart_access_brands_group.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'smart_access_brands_validation_group'
+require_relative 'smart_access_brands_retrieval_group'
+
+module SMARTAppLaunch
+  class SMARTAccessBrandsGroup < Inferno::TestGroup
```

Comment:
I think we want this as a `run_as_group`?  I'm not sure it makes much sense to allow users to run the individual groups, as the second relies on the first having been run first, right?

It is nice and defensive, but I'm not sure what the value is.  Plus, the Service Base URL tests are 'run_as_group'.

<img width="917" alt="Screenshot 2024-10-01 at 4 43 12 PM" src="https://github.com/user-attachments/assets/763bc028-91a9-4b16-ae64-22091cf7634b">